.gitignore

```diff
@@ -20,3 +20,4 @@ debian/pylint.substvars
 debian/pylint
 .coverage
 .coverage.*
+.idea
```

Comment:
This could be moved in a global `.gitignore` file though

Reply:
It could, but some projects include certain files from the .idea directory and that would interfere with those.  
https://intellij-support.jetbrains.com/hc/en-us/articles/206544839